src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -68,40 +95,53 @@
+          [comps/TabBar {:selected-index (tab-string-to-index tab)
+                         :items
+                         [{:text SUMMARY :href (nav/create-href (:nav-context props))
+                           :content
+                           (react/create-element
+                             [summary-tab/Summary {:key workspace-id :ref SUMMARY
+                                                   :workspace-id workspace-id
+                                                   :bucket-access? (:bucket-access? @state)
+                                                   :nav-context nav-context
+                                                   :on-delete (:on-delete props)
+                                                   :on-clone (:on-clone props)}])
+                           :onTabRefreshed #(react/call :refresh (@refs SUMMARY))}
+                          {:text DATA :href (nav/create-href (:nav-context props) DATA)
+                           :content
+                           (react/create-element
+                             [data-tab/WorkspaceData {:ref DATA :workspace-id workspace-id}])
+                           :onTabRefreshed #(react/call :refresh (@refs DATA))}
+                          {:text CONFIGS :href (nav/create-href (:nav-context props) CONFIGS)
+                           :content
+                           (react/create-element
+                             [method-configs-tab/Page {:ref CONFIGS
+                                                       :workspace-id workspace-id
+                                                       :bucket-access? (:bucket-access? @state)
+                                                       :on-submission-success #(nav/navigate (:nav-context props) MONITOR %)
+                                                       :nav-context (nav/terminate-when (not= tab CONFIGS) nav-context)}])
+                           :onTabRefreshed #(react/call :refresh (@refs CONFIGS))}
+                          {:text MONITOR :href (nav/create-href (:nav-context props) MONITOR)
+                           :content
+                           (react/create-element
+                             [monitor-tab/Page {:ref MONITOR
+                                                :workspace-id workspace-id
+                                                :nav-context (nav/terminate-when (not= tab MONITOR) nav-context)}])
+                           :onTabRefreshed #(react/call :refresh (@refs MONITOR))}]}]]))
+   :component-did-mount
+   (fn [{:keys [props state]}]
+    (endpoints/call-ajax-orch
+      {:endpoint (endpoints/get-workspace-bucket (:workspace-id props))
+       :on-done (fn [{:keys [success? status-code get-parsed-response status-text]}]
+                    (if success?
+                      (swap! state assoc :status :success :status-code status-code :bucket-access? true)
+                      (swap! state assoc :status :success :status-code status-code :bucket-access? false)))}))})
```

Comment:
Looks like you're not using `:status`, you're tracking the result of this AJAX call through `:bucket-access?`.  The whole `if` can be shortened to:

    (swap! state assoc :status-code status-code :bucket-access? success?)